src/sdk/dto/fetch-exchange-rates.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { IsAddress } from './validators';
+import { ArrayNotEmpty, IsInt, IsPositive } from 'class-validator';
+
+export class FetchExchangeRatesDto {
+  @IsAddress({
+    each: true,
+  })
```

Comment:
Should there be a space here?

Reply:
No, As per formatting there is no space.